sentry/README.md

```diff
@@ -55,3 +55,4 @@ Parameter                          | Description
 `metrics.serviceMonitor.honorLabels` | honorLabels chooses the metric's labels on collisions with target labels. | `false`
 `metrics.serviceMonitor.namespace` | namespace where servicemonitor resource should be created | `the same namespace as sentry`
 `metrics.serviceMonitor.scrapeInterval` | interval between Prometheus scraping | `30s`
+`sentry.secretKey` | **Required** secret key for the session cookie ([documentation](https://develop.sentry.dev/config/#general)) | `nil`
```

Comment:
I think you can remove *required*.

Also, could you add a specific note to the readme ?
Similar to https://github.com/sentry-kubernetes/charts#postgressql